server/constants/vars.js

```diff
@@ -47,3 +61,9 @@ export const ALL_SUBJECTS_MONGO_PROJECTION = {
   _id: 0,
   subject: 1,
 }
+
+export const INDIVIDUAL_FILTERS_MONGO_PROJECTION = {
```

Comment:
is this used in multiple places? It _seems_ specific to the 1 Mongo query. If so, I think we should keep it isolated to that module.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/280/commits/81cad2058d15f6a15a126abcecbb091090e31058